examples/vm_scheduling/reinforcement_learning/common/components/state.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import numpy as np
+import pandas as pd
+
+
+PM_ATTRIBUTES = ["cpu_cores_capacity", "memory_capacity", \
```

Comment:
The `PM_ATTRIBUTES` here are not aligned with the `VM_ATTRIBUTES`.
The previous one is used to extract features from the snapshot_list, but the latter one is the list of features used.

Reply:
I will add PM_EXTRACTED_ATTRIBUTES to represent the features extracted from the snapshot_list, and PM_ATTRIBUTES to represent the list of features used.